reemgawad/theatr
# Video activity

- Added a _video_activity.html.erb
- the activity with quiz category use the same template since the only difference is a video
- added a migration for a video:string to activities table
- Added the video activity to the quiz

to test:

migrate then seed, add the video activity to the classroom as well as a quiz activity and see if they appear and see if you can fill them
